src/logic/lint.ts

```diff
@@ -161,13 +163,18 @@ export function formatHuman({
         }
       } else {
         buffer += color ? red(formatIssues([error])) : formatIssues([error]);
+        
+        if (report.errors.length > 0) {
+          buffer += '\n';
+        }
```

Comment:
Why do you have this here? Also why in `else` branch?

Reply:
formatIssues joins issues with newline character, but only when there are more then one issue, therefore these example errors were not separated.